programmers/알고리즘_3기/2주차/자연수-뒤집어-배열로-만들기.js

```diff
@@ -0,0 +1,20 @@
+function solution(n) {
+  let newN = String(n).split("").reverse();
```

Comment:
newN이라는 값은 변하지 않는 값이니까 let 대신 const가 좋아요!
```suggestion
  const newN = String(n).split("").reverse();
```